source/custom/nixnet_converters.h

```diff
@@ -106,7 +106,7 @@ struct FrameHolder {
         throw std::invalid_argument("All FrameBufferRequest instances in repeated field should have same oneof set for the frame");
       }
 
-      u32 frame_size = get_frame_buffer_size(1, grpc_frame.enet().frame_data().length(),Protocol::PROTOCOL_ENET);
+      u32 frame_size = get_frame_buffer_size(1, static_cast<u32>(grpc_frame.enet().frame_data().length()), Protocol::PROTOCOL_ENET);
```

Comment:
This just cleans up some build warnings I had noticed. Not directly related to this PR.